packages/components/src/vue/components/feature-settings.vue

```diff
@@ -72,6 +72,12 @@
           @change="featureOptionChanged"
       >
       </setting>
+      <setting
+          class="alpheios-feature-options__item"
+          :data="featureOptions.items.enableMouseMoveLimitedByIdCheck"
+          @change="featureOptionChanged"
+      >
+      </setting>
```

Comment:
I removed the second parameter with id selector only from publishing here, but left it with default value in "private" mode.

Reply:
see above